data-jdbc/src/test/resources/application-oracle-jsonview.yml

```diff
@@ -14,4 +14,5 @@ datasources:
   default:
     db-type: oracle
     dialect: oracle
-    schema-generate: NONE
+    schema-generate: VALIDATE
```

Comment:
We have this test where tables are created by Flyway and it is triggered before schema validation so this is good case for test.
Other tests are manual table creation and other tests for dialects for tables creates using `SchemaGenerator`